packages/compass-indexes/src/components/usage-column/usage-column.jsx

```diff
@@ -43,43 +28,30 @@ class UsageColumn extends PureComponent {
       return null;
     }
     return (
-      <div className={classnames(styles['usage-column-since'])}>
+      <span>
         since&nbsp;
-        <span>
-          {this.props.since ? this.props.since.toDateString() : 'N/A'}
-        </span>
-      </div>
+        {this.props.since ? this.props.since.toDateString() : 'N/A'}
+      </span>
     );
   }
 
-  /**
-   * Render the usage column.
-   *
-   * @returns {React.Component} The usage column.
-   */
   render() {
     const usage = isUndefined(this.props.usage) ? '0' : this.props.usage;
-    const tooltipText = `${this.tooltip()}`;
-    const tooltipOptions = {
-      'data-tip': tooltipText,
-      'data-for': TOOLTIP_ID,
-      'data-effect': 'solid',
-      'data-border': true,
-      'data-multiline': true,
-    };
+    const tooltip = this.tooltip();
     return (
-      <td className={classnames(styles['usage-column'])}>
-        <span>
-          <div
-            {...tooltipOptions}
-            className={classnames(styles['usage-column-quantity'])}
-            data-test-id="index-table-usage"
-          >
-            <ReactTooltip id={TOOLTIP_ID} />
-            {usage}
-          </div>
-          {this.renderSince()}
-        </span>
+      <td>
+        <Tooltip
+          trigger={({ children, ...props }) => (
+            <span {...props}>
+              {children}
+              <Body>
+                {usage}&nbsp;({this.renderSince()})
```

Comment:
Do we want to hide the `()` around the `renderSince` date when there isn't any usage:
![Screen Shot 2022-08-15 at 4 52 50 PM](https://user-images.githubusercontent.com/1791149/184716391-ec9ec902-2bf8-4c2d-9d78-0707ba5921aa.png)